src/lib/routes/admin-api/metrics.ts

```diff
@@ -187,18 +187,24 @@ class MetricsController extends Controller {
         }
         try {
             const hoursToQuery = 6;
-            const [clientMetrics, adminMetrics] = await Promise.all([
-                this.clientInstanceService.getRPSForPath(
-                    '/api/client/.*',
-                    hoursToQuery,
-                ),
-                this.clientInstanceService.getRPSForPath(
-                    '/api/admin/.*',
-                    hoursToQuery,
-                ),
-            ]);
+            const [clientMetrics, frontendMetrics, adminMetrics] =
+                await Promise.all([
+                    this.clientInstanceService.getRPSForPath(
+                        '/api/client/.*',
+                        hoursToQuery,
+                    ),
+                    this.clientInstanceService.getRPSForPath(
+                        '/api/frontend.*',
+                        hoursToQuery,
+                    ),
```

Comment:
Adds `frontend` (embedded proxy) requests to the metrics.

Reply:
Nice. Gaston I talked about it, but didn't get around to do it just yet.